recipes/sdl/all/conandata.yml

```
@@ -1,4 +1,7 @@
 sources:
+  "2.30.3":
+    url: "https://www.libsdl.org/release/SDL2-2.30.3.tar.gz"
```

```suggestion
    url: 
      - "https://github.com/libsdl-org/SDL/releases/download/release-2.30.3/SDL2-2.30.3.tar.gz"
      - "https://www.libsdl.org/release/SDL2-2.30.3.tar.gz"
```

I would like to add GitHub as a mirror

---

I like this, thanks or the suggestion!